hpedockerplugin/hpe/hpe_3par_common.py

```diff
@@ -165,10 +166,17 @@ def check_flags(self, options, required_flags):
                 LOG.error(msg)
                 raise exception.InvalidInput(reason=msg)
 
-    def _create_client(self):
-        cl = client.HPE3ParClient(
-            self.config.hpe3par_api_url,
-            suppress_ssl_warnings=CONF.suppress_requests_ssl_warnings)
+    def _create_client(self, timeout=None):
```

Comment:
Please remove default value for timeout from here. Let's enforce this parameter on the caller as in production environment it has been recommended to use this value.